com.avaloq.tools.ddk.check.ui.test/src/com/avaloq/tools/ddk/check/ui/test/quickfix/CheckQuickfixTest.xtend

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016 Avaloq Evolution AG and others.
+ * Copyright (c) 2016-2018 Avaloq Evolution AG and others.

```

Comment:
There seems to be some inconsistency with these. The default templates leave this as 2016. Do we really care about the years that much? If yes, we need to update the whole repo, if no, then revert this line and use the template in other new sources.

Reply:
All changed to 2016. (Which is incorrect but consistent!)